### DISCUSSION TITLE
How fix it?

### DISCUSSION THREAD

**Discussion by REFGOLD** [FlareSolverr/FlareSolverr#577] [General]
16gb RAM
![image](https://user-images.githubusercontent.com/45120499/200635540-9fbbd369-62aa-41e0-a5a9-8683856afaba.png)

![image](https://user-images.githubusercontent.com/45120499/200323018-08724517-23af-4783-9504-91bfd69166cd.png)

**Comment by ngosang** [comment]
FlareSolverr 3.0.0 is out. You won't see this issue anymore.
